cypress/integration/portal-report-smoke-test.spec.js

```diff
@@ -20,7 +20,9 @@ context("Portal Report Smoke Test", () => {
     // run, so this doesn't seem necessary.
 
     beforeEach(() => {
-        cy.visit("/");
+      let fakeServer = "http://portal.test";
+
+        cy.visit(`/?reportUrl=${encodeURIComponent(fakeServer)}`);
```

Comment:
The reportUrl param is deprecated in the portal, so it shouldn't be used here in the tests.